src/AppInstallerCLITests/TestSource.cpp

```diff
@@ -181,6 +183,36 @@ namespace TestCommon
         return false;
     }
 
+    bool TestPackage::IsSame(const IPackage* other) const
+    {
+        const TestPackage* otherAvailable = dynamic_cast<const TestPackage*>(other);
+
+        if (otherAvailable)

```

Comment:
>if (otherAvailable) [](start = 8, length = 19)

nit: maybe return false early if null so we have a clearer flow(avoiding return true between return falses)

Reply:
Update this and other cases to be much more terse.